src/Raven.Studio/typescript/components/pages/database/documents/identities/DocumentIdentities.stories.tsx

```diff
@@ -28,7 +28,11 @@ export const DocumentIdentitiesStory: StoryObj<DocumentIdentitiesStoryArgs> = {
 
         databasesService.withIdentities(args.identities);
 
-        return <DocumentIdentities />;
+        return (
+            <div className="h-100">
+                <DocumentIdentities />
+            </div>
+        );
```

Comment:
We should wrap all stories that way. Add this class maybe here
https://github.com/ravendb/ravendb/blob/b46a627fc2196caf732a1c53558823f8531d764a/src/Raven.Studio/.storybook/preview.js#L78

Reply:
Done ✅